www/include/reporting/dashboard/template/viewHostLog.ihtml

```diff
@@ -24,8 +22,14 @@
 				<input type="radio" value="preset" name="period_choice" {if $period_choice == "preset" } checked="checked" {/if}/>
 					{$periodTitle} : {$formPeriod.period.html}{$periodORlabel}
 				<input type="radio" name="period_choice" value="custom" {if $period_choice == "custom" } checked="checked" {/if} />
-					{$formPeriod.StartDate.label} {$formPeriod.StartDate.html}&nbsp;{$formPeriod.alternativeDateStartDate.html}
-					{$formPeriod.EndDate.label} {$formPeriod.EndDate.html} {$formPeriod.alternativeDateEndDate.html}
+					{$formPeriod.StartDate.label} {$formPeriod.StartDate.html}&nbsp;
+					{if isset($formPeriod.alternativeDateStartDate)}
+						{$formPeriod.alternativeDateStartDate.html}
```

Comment:
this one is mandatory to be able to localize the date.
Shouldn't you set it if not set ?